ingredient/ingredient-storage/src/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
move this file up to project root, and fix include path to point to src.

Reply:
Moved tsconfig and fixed references.